mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -150,14 +155,14 @@ class LinearRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
        as.numeric.data.V2. 6.995908
        as.numeric.data.V3. 5.275131
      */
-    val weightsR = Vectors.dense(6.995908, 5.275131)
+      val weightsR = Vectors.dense(6.995908, 5.275131)
 
-    assert(model1.intercept ~== 0 absTol 1E-3)
-    assert(model1.weights ~= weightsR relTol 1E-3)
-    assert(model2.intercept ~== 0 absTol 1E-3)
-    assert(model2.weights ~= weightsR relTol 1E-3)
+      assert(model1.intercept ~== 0 absTol 1E-3)
```

Comment:
These lines become inside of `foreach` block. So these are not indentation issues.